iot-hub/Samples/service/JobsSample/JobsSample.cs

```diff
@@ -57,7 +57,7 @@ public async Task RunSampleAsync()
             // *************************************** Get all Jobs ***************************************
             IEnumerable<JobResponse> queryResults = await _jobClient.CreateQuery().GetNextAsJobResponseAsync().ConfigureAwait(false);
 
-            List<JobResponse> getJobs = queryResults.ToList();
+            var getJobs = queryResults.ToList();

```

Comment:
Type not evident from a call to `ToList()`. What type of elements are in the list? Keep the explicit type.